### PR TITLE
fix(client): form editor emits attach and detach events

### LINKED ISSUE #2197: Moving a text field below a button results in error
__Describe the Bug__

Given the following example form

```json
{
  "type": "default",
  "components": [
    {
      "key": "textfield1",
      "label": "Text Field",
      "type": "textfield"
    },
    {
      "key": "textfield2",
      "label": "Text Field",
      "type": "textfield"
    },
    {
      "key": "button6",
      "label": "Button",
      "type": "button"
    }
  ]
}
```

and moving the first text field below the button, the text field got erased from the editor. Also, an error is displayed in the Developer Console.

Kapture 2021-04-08 at 14 41 32

image

__Steps to Reproduce__

1. Open example form from above
2. Drag & drop the first text field to the very bottom
3. Recognize the text field is lost
4. Notice the text field is still persisted in the JSON view

__Expected Behavior__

Moving the text field should work without problems.

__Environment__

- OS: MacOS 10.15
- Camunda Modeler Version: 4.7.0-rc.0

### PR DESCRIPTION
**Pull request by philippfromme**
Fixes drag and drop interaction in the form editor and allows drop across full height.

Preview:

Ts3zV2kQwm

Requires https://github.com/bpmn-io/form-js/pull/51 to be integrated.

Closes #2197

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "@bpmn-io/align-to-origin": "^0.7.0",
     "@bpmn-io/dmn-migrate": "^0.4.3",
     "@bpmn-io/extract-process-variables": "^0.4.3",
-    "@bpmn-io/form-js": "0.0.10",
+    "@bpmn-io/form-js": "0.0.11",
```
Comment (barmac) [inline client/package.json:13]: any plans to release 0.1.0? :P

**Review by barmac — CHANGES_REQUESTED** [review]
For some reason I cannot set `required` in this version.

**Comment by philippfromme** [comment]
@barmac Will be addressed by https://github.com/bpmn-io/form-js/pull/53.

File: client/package.json (hunk 1 above)
Comment (nikku) [reply client/package.json]: No.

Comment (nikku) [reply client/package.json]: What would be a compeling reason?

Comment (philippfromme) [reply client/package.json]: Agreed. There's nothing to win at the moment.

**Comment by philippfromme** [comment]
@nikku Can you take over this review?

**Comment by nikku** [comment]
Will do!

**Comment by philippfromme** [comment]
😍

**Review by nikku — APPROVED** [review]

**Comment by nikku** [comment]
Works for me.

**Pull request merged**